### PR TITLE
update deps, add sbt-updates to build build

### PR DESCRIPTION
**Pull request by tpolecat**
This updates Flyway and http4s in the main project, adds sbt-updates to the metabuild, and updates some plugins there. It also removes explicit Scala version settings because these are defined by sbt-gsp.

### CODE REVIEW

**Comment by tpolecat** [comment]
I'm going to wait until sbt-gsp and gsp-math deps trickle down.

**Comment by tpolecat** [comment]
Ok @cquiroz see how this looks.

File: build.sbt [hunk 1]
```diff
@@ -4,19 +4,15 @@ import sbtcrossproject.CrossType
 lazy val doobieVersion           = "0.8.6"
 lazy val fs2Version              = "2.2.2"
 lazy val geminiLocalesVersion    = "0.2.0"
-lazy val gspMathVersion          = "0.1.13"
+lazy val gspMathVersion          = "0.1.14"
 lazy val kindProjectorVersion    = "0.11.0"
-lazy val monocleVersion          = "2.0.1"
+lazy val monocleVersion          = "2.0.2"
```
Comment (cquiroz) [inline build.sbt]: 👍

File: build.sbt [hunk 2]
```diff
@@ -4,19 +4,15 @@ import sbtcrossproject.CrossType
 lazy val doobieVersion           = "0.8.6"
 lazy val fs2Version              = "2.2.2"
 lazy val geminiLocalesVersion    = "0.2.0"
-lazy val gspMathVersion          = "0.1.13"
+lazy val gspMathVersion          = "0.1.14"
```
Comment (cquiroz) [inline build.sbt]: I guess this should be 0.1.15

Comment (tpolecat) [reply build.sbt]: Doh! Done.

File: build.sbt [hunk 3]
```diff
@@ -48,7 +44,6 @@ inThisBuild(Seq(
   homepage := Some(url("https://github.com/gemini-hlsw/gsp-core")),
   addCompilerPlugin("org.typelevel" % "kind-projector" % kindProjectorVersion cross CrossVersion.full),
   resolvers += "Gemini Repository" at "https://github.com/gemini-hlsw/maven-repo/raw/master/releases", // for gemini-locales
-  crossScalaVersions := Seq(scala212Version, scala213Version),
```
Comment (cquiroz) [inline build.sbt]: Wouldn't this publish or 2.13 only?

Comment (tpolecat) [reply build.sbt]: No, this setting comes for free from sbt-gsp.

**Pull request merged**